src/RefractAST.h

```diff
@@ -2,27 +2,19 @@
 //  RefractAST.h
 //  drafter
 //
-//  Created by Jiri Kratochvil on 18/05/15.
+//  Created by Jiri Kratochvil on 31/07/15.
```

Comment:
Is there a reason this creation date is changing? Also, why is this in the file when you can find the info via git?